vignettes/ChromBand.Rmd

```diff
@@ -0,0 +1,379 @@
+---
+title: "Using Chromosome Bands as Categories"
+author: 
+   - name: D. Sarkar
+   - name: S. Falcon
+   - name: R. Gentleman
+   - name: Kemmy MO Jones
+     affiliation: [ Chromosome Bands Vignette translation from Sweave to R 
+     Markdown / HTML ]
+date: "`r format(Sys.Date(), '%B %d, %Y')`"
+package: Category
+vignette: >
+  %\VignetteIndexEntry{Using Chromosome Bands as Categories}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document:
+             toc: false
```

Comment:
Fix the indent. YAML indents 2 spaces at each level, so there should be 6 spaces in front of `toc`.

Reply:
But that is the syntax of YAML according to their documentation, because the doc is extended from the document, please check the YAML documents you gave us.